specs/helper.js

```diff
@@ -194,11 +194,11 @@ const dispatchMockKeyDownEvent = dispatchMockEvent(Simulate.keyDown);
 /**
  * Dispatch an 'esc' key down event from an element.
  */
-export const escKeyDown = dispatchMockKeyDownEvent("ESC", 27);
+export const escKeyDown = dispatchMockKeyDownEvent("ESC", "Escape");
 /**
  * Dispatch a 'tab' key down event from an element.
  */
-export const tabKeyDown = dispatchMockKeyDownEvent("TAB", 9);
+export const tabKeyDown = dispatchMockKeyDownEvent("TAB", "Tab");
```

Comment:
Here we need to duplicate both dispatchers `escKeyDownWithCode` and `tabKeyDownWithCode`

export const tabKeyDown = dispatchMockKeyDownEvent("TAB", { code: "Tab" });

Reply:
```suggestion
export const tabKeyDown = dispatchMockKeyDownEvent("TAB", { keyCode: 9 });
```

Reply:
If we duplicate the dispatchers, do you suggest also duplicating all tests that use them to cover both the legacy `KeyboardEvent.keyCode` and `KeyboardEvent.code`? I started doing this locally but there's quite a lot, so wanted to make sure that's what you have in mind.

Reply:
Looks good.